Switchcraft/Classes/Switchcraft.swift

```diff
@@ -14,6 +14,11 @@ public protocol SwitchcraftDelegate: AnyObject {
      * Called when an endpoint is selected.
      */
     func switchcraft(_ switchcraft: Switchcraft, didSelectEndpoint endpoint: Endpoint)
+
+    /**
+     * Called when an action is tapped.
+     */
+    func switchcraft(_ switchcraft: Switchcraft, didTapAction action: Action)
```

Comment:
@robmaceachern  or @nbrooke might be able to help me out here.

I expect the average user won't need to use this method, and AFAICT you can't have optional protocol functions in Swift, and our protocols can't be marked `@objc` without changing them completely (or at all?).

I see a couple different options:
1. Just leave this non-optional, as-is
2. Add a second delegate, something like `SwitchcraftActionDelegate` and put this there
3. Figure out how to mark the delegate functions as `@obj-c` so we can use `optional`
4. Something I'm missing?

Any thoughts on what the best course might be?

Reply:
I think you can add a no-op default implementation which can be overridden if needed. See "Provide a default implementation" in https://useyourloaf.com/blog/swift-optional-protocol-methods/